### ISSUE TITLE
New Rule: no-blanks-blockquote

### ISSUE THREAD

**Issue by ntdiary** [eslint/markdown#388]
### Rule details

This rule is triggered when two blockquote blocks are separated by nothing except for a blank line

### What type of rule is this?

Warns about a potential problem

### Example code

```markdown
> one blockquote

> same blockquote?

If they are the supposed to be the same quote, then add the blockquote symbol at the beginning of the blank line:
> one blockquote
> 
> the same blockquote

If not, ensure there is appropriate explanatory text between them:

> This is a blockquote.

And Jimmy also said:

> This is another blockquote.
```

### Participation

- [x] I am willing to submit a pull request to implement this rule.

### Additional comments

Ref: [MD028/no-blanks-blockquote](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md028---blank-line-inside-blockquote)

may be considered an edge case.

**Comment by nzakas** [comment]
I'd like to get a reproduction case where we can see this error in action before deciding what to do. It may true that different parsers do different things and this seems like a pretty extreme edge case that may not warrant a rule.

**Comment by snitin315** [comment]
Thank you for raising this issue. However, due to the lack of follow-up and insufficient information provided, we are unable to proceed further. If you have more details or updates, please feel free to reopen the issue or create a new one.
